WolvenKit.Modkit/RED4/Import.cs

```diff
@@ -633,14 +637,31 @@ void SetTextureGroupSetup(STextureGroupSetup setup, CR2WFile cr2w)
 
         private bool ImportGltf(RedRelativePath rawRelative, DirectoryInfo outDir, GltfImportArgs args)
         {
+            var am = Locator.Current.GetService<IArchiveManager>();
+            var somefile = am.Lookup(11327483717126307343); //some ice cube  11327483717126307343
+                                                            //some cube     15966709641681476693
```

Comment:
???

Reply:
uses the ice cube mesh as default if no mesh is provided, planning to polish that feature a bit more